client/src/pages/ViewContestSubmissions/ViewContestSubmissions.tsx

```diff
@@ -0,0 +1,114 @@
+import react, { useState, useEffect } from 'react';
+import { Grid, Typography, Box, Chip, Avatar } from '@material-ui/core';
+import GridList from '@material-ui/core/GridList';
+import GridListTile from '@material-ui/core/GridListTile';
+import GridListTileBar from '@material-ui/core/GridListTileBar';
+import IconButton from '@material-ui/core/IconButton';
+import AddCircleOutlineIcon from '@material-ui/icons/AddCircleOutline';
+import useStyles from './useStyles';
+
+import { contest, contests } from '../../interface/tempContestData';
```

Comment:
```suggestion
import { Contest, Contests } from '../../interface/tempContestData';
```
Would be better practice to have these interfaces capitalized